darts/models/forecasting/forecasting_model.py

```diff
@@ -2106,6 +2115,57 @@ def _optimized_historical_forecasts(
         )
         return []
 
+    @staticmethod
+    def _inverse_transform_forecasts_list(
+        forecasts_list_transformed: Union[List[TimeSeries], List[List[TimeSeries]]],
+        scaler: Scaler,
+    ) -> Union[List[TimeSeries], List[List[TimeSeries]]]:
+        """Applies inverse transform on the list (of lists) of forecasts.
+        Uses the scaler trained previously on the same set of series.
+
+        Parameters
+        ----------
+        forecasts_list_transformed
+            The list (of lists) of forecasts.
+        scaler
+            The scaler trained on the same set of series that was used to generate forecasts.
+
+        Returns
+        -------
+        Union[List[TimeSeries], List[List[TimeSeries]]]
+            A list (of lists) of inversely transformed forecasts.
+
+        """
+
+        if not forecasts_list_transformed:
+            return forecasts_list_transformed
+
+        if isinstance(forecasts_list_transformed[0], TimeSeries):
+            forecasts_list_inversely_transformed = scaler.inverse_transform(
+                forecasts_list_transformed
+            )
+        else:
+            fill_value = TimeSeries.from_values(np.empty(1))
+
+            forecasts_list_inversely_transformed = list(
```

Comment:
at this stage here the list is not inversely transformed. Perhaps another name for this variable.